cisco_aci/datadog_checks/cisco_aci/fabric.py

```diff
@@ -209,3 +243,71 @@ def get_fabric_type(self, obj_type):
             return 'pod'
         if obj_type == 'l1PhysIf':
             return 'port'
+
+    def batch_payloads(self, devices, interfaces, collect_ts):
+        for device in devices:
+            yield NetworkDevicesMetadata(namespace=self.namespace, devices=[device], collect_timestamp=collect_ts)
+
+        payloads = []
+        for interface in interfaces:
+            if len(payloads) == PAYLOAD_METADATA_BATCH_SIZE:
+                yield NetworkDevicesMetadata(
+                    namespace=self.namespace, interfaces=payloads, collect_timestamp=collect_ts
+                )
+                payloads = []
+            payloads.append(interface)
+        if payloads:
+            yield NetworkDevicesMetadata(namespace=self.namespace, interfaces=payloads, collect_timestamp=collect_ts)
+
+    def submit_node_metadata(self, node_attrs, tags):
+        node = Node(attributes=node_attrs)
+        id_tags = ['namespace:{}'.format(self.namespace), 'system_ip:{}'.format(node.attributes.address)]
+        device_tags = [
+            'device_vendor:{}'.format(VENDOR_CISCO),
+            'device_namespace:{}'.format(self.namespace),
+            'device_hostname:{}'.format(node.attributes.dn),
+            'hostname:{}'.format(node.attributes.dn),
+            'system_ip:{}'.format(node.attributes.address),
```

Comment:
is `system_ip` a common Cisco ACI term? thought this was cisco sd-wan specific

Reply:
oop - i think i was referencing your PR for some id tags and accidentally incl. that i can remove! slipped me!